library/ZendService/Twitter/Response.php

```diff
@@ -152,4 +160,52 @@ public function toValue()
     {
         return $this->jsonBody;
     }
+
+    /**
+     * Retun the array of response rate info
+     *
+     * @return array
+     */
+    public function getRateLimit()
+    {
+        return $this->rateLimit;
+    }
+
+    /**
+     * Populates the object with info. This can possibly called from the
```

Comment:
Hi! :)

I am confused by the comment. getRateLimit has nothing to do with the __construct or the httpResponse. It's sole purpose it to allow read-only access tot he protected array rateLimit.  

Can you please clarify?

Cheers!
=C=